pkg/spec/postgresql.go

```diff
@@ -252,22 +280,16 @@ func (p *Postgresql) UnmarshalJSON(data []byte) error {
 	}
 	tmp2 := Postgresql(tmp)
 
-	clusterName, err := extractClusterName(tmp2.ObjectMeta.Name, tmp2.Spec.TeamID)
-	if err == nil {
-		tmp2.Spec.ClusterName = clusterName
-	} else {
+	if clusterName, err := extractClusterName(tmp2.ObjectMeta.Name, tmp2.Spec.TeamID); err != nil {
 		tmp2.Error = err
 		tmp2.Status = ClusterStatusInvalid
+	} else if err := validateCloneClusterDescription(&tmp2.Spec.Clone); err != nil {
+		tmp2.Error = err
+		tmp2.Status = ClusterStatusInvalid
+	} else {
```

Comment:
i feel the summary of [this discussion](https://github.com/zalando-incubator/postgres-operator/pull/270#issuecomment-379200852) is worth a comment here:

> anyone able to deploy a manifest can clone from any cluster's backups, and thus read all the data in other clusters.
>> It pretty much depends on how different teams are separated on the infrastructure level;

i.e. it is outside of the operator scope to check this